include/ice/ssram.h

```diff
@@ -1,19 +1,45 @@
-/** Low-level API for communicating with flash chips: reading and writing.
+/** Low-level API for communicating with serial memory chips: reading and writing.
 * \defgroup pico_ice_ssram
 */
 
 #ifndef _ICE_SSRAM_H
 #define _ICE_SSRAM_H
 
+#define ICE_SSRAM_STATUS_BUSY_MASK  0x01
```

Comment:
Is there anywhere where this is used? Or is it anticipating future change?

Reply:
It would be used in combination with ice_ssram_get_status() to determine if a flash write or erase has completed. Perhaps a "higher-level" API function could be added specifically to do that.